CMakeLists.txt

```diff
@@ -1,13 +1,14 @@
 cmake_minimum_required(VERSION 3.5)
 
-project(s2-geometry)
+project(s2geometry VERSION 0.9.0)
 
 include(CMakeDependentOption)
 include(CheckCXXCompilerFlag)
 include(FeatureSummary)
 include(FindPackageHandleStandardArgs)
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Is this necessary and why? If it is, add a comment.

Reply:
It's not necessary. Dropped.